Source/Tools/FEXLoader/LinuxSyscalls/x32/Ioctl/drm.h

```diff
@@ -7,6 +7,7 @@
 
 #include <cstdint>
 extern "C" {
+#define __user
```

Comment:
It's mentioned that this is needed, but what is it actually used for and where is it normally defined?

Reply:
It's used in the drm headers themselves. I previously modified them all to remove the attribute but that modification has been the biggest change for every drm header uprev so I've stopped so we can use them unmodified from upstream.

Reply:
Added a comment.